hr_employee_birth_name/readme/DESCRIPTION.rst

```diff
@@ -0,0 +1,3 @@
+This module adds the following to employees :
```

Comment:
Coul you take the opportunity to extend it a bit?

Reply:
Hello @tafaRU 

This is a migration, that could be improved in other PR.